src/ImageStats/Histogram.cc

```diff
@@ -94,3 +94,18 @@ void Histogram::SetHistogramBins(const std::vector<int>& bins) {
     }
     _histogram_bins = bins;
 }
+
+bool Histogram::HasDominantBin() {
```

Comment:
I think `HasDominantBin` sounds like it should be a `const` function, but it isn't. Perhaps `CheckForDominantBin`?